crates/rooch-rpc-api/src/api/rooch_api.rs

```diff
@@ -41,9 +41,6 @@ pub trait RoochAPI {
         tx_option: Option<TxOptions>,
     ) -> RpcResult<ExecuteTransactionResponseView>;
 
-    #[method(name = "dryRunRawTransaction")]
-    async fn dry_run(&self, tx_bcs_hex: BytesView) -> RpcResult<DryRunTransactionResponseView>;
-
```

Comment:
We should keep this API for the web DApp.

Reply:
OK, i will add the removed API.